Makefile

```diff
@@ -8,9 +8,15 @@ build:
 clean:
 	docker-compose down -v --remove-orphans
 
+debug:
+	docker-compose kill app ; docker-compose -f docker-compose.debug.yml -f docker-compose.yml run --service-ports app
```

Comment:
We really shouldn't ever be sending sigkill. It could cause the application to terminate in an unexpected way, leaving the containers in an inconsistent state, and causing us to debug problems that don't exist. A real headache that we should avoid.

Reply:
For this a docker-compose stop would work. You could use kill -s SIGINT, but I think stop is better for what we're doing.

Reply:
Moved to stop functionality